Percy/Percy.cs

```diff
@@ -245,38 +262,48 @@ public static void Screenshot(WebDriver driver, string name, IEnumerable<KeyValu
                             userOptions["consider_region_elements"] = elementIds;
                         }
                     }
+                    userOptions[SYNC] = isSync;
+                    if(userOptions.ContainsKey(SYNC)) {
+                        isSync = (bool)userOptions[SYNC];
+                        
+                    }
                     screenshotOptions.Add("options", userOptions);
                 }
 
                 dynamic res = Request("/percy/automateScreenshot", JObject.FromObject(screenshotOptions), true);
                 dynamic data = JsonSerializer.Deserialize<object>(res.content);
                 if (data.GetProperty("success").GetBoolean() != true)
                     throw new Exception(data.GetProperty("error").GetString());
+                if (isSync) {
+                    return JObject.FromObject(data.GetProperty("data"));
+                }
+                return null;
             }
             catch(Exception error)
             {
                 Log($"Could not take Percy Screenshot \"{name}\"");
                 Log(error);
+                return null;
             }
         }
 
-        public static void Snapshot(WebDriver driver, string name, object opts)
+        public static JObject Snapshot(WebDriver driver, string name, object opts, bool sync = false)
```

Comment:
move sync inside options + no default value for sync

Reply:
done